spec/support/shared_csv_export_behavior.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
As "export" is a core Active Scaffold action, I would move these examples to `spec/support/shared_active_scaffold_page_behavior.rb` and delete this file.

Reply:
Export is not a core active scaffold action. We pull in the active_scaffold_export gem to add it, and the fact that is in a separate gem is *exactly* why it's in a different shared behavior.

Moved into the active scaffold helper nonetheless.